type_data/tables/armor.yaml

```diff
@@ -4,6 +4,9 @@ name:
   - .*?chain mail
   - .*?half plate
   - .*?augmented chain
+exclude:
```

Comment:
I couldn't add these through a migration because the table didn't previously have an `exclude` key which caused my migration to fail validations. I think this is a reasonable validation to include but it might make sense to always have all three keys (name, noun, and exclude) present in the yaml files even if they have empty values.